src/sw-entry.ts

```diff
@@ -0,0 +1,5 @@
+if ('serviceWorker' in navigator) {
```

Comment:
Is this file needed now?

Reply:
It is not; it's in `@dojo/webpack-contrib` now. Good catch!